src/instruments/src/SD/Pages/Hyd/Hyd.tsx

```diff
@@ -0,0 +1,776 @@
+                <line className="GreenLine Hide" x1={290} y1={y + 180} x2={300} y2={y + 180} />
+                <Triangle x={290} y={y + 180} colour="White" fill={0} orientation={90} />
+
+                <text
+                    id="ELEC-centre"
+                    className={!ACBus1IsPowered ? 'RatPtuElec FillAmber' : 'RatPtuElec FillWhite'}
+                    x={350}
+                    y={y + 245}
+                    alignmentBaseline="central"
+                >
+                    ELEC
+
+                </text>
+
+                <text
+                    id="ELEC-right"
+                    className={!ACBus2IsPowered ? 'RatPtuElec FillAmber' : 'RatPtuElec FillWhite'}
+                    x={548}
+                    y={y + 180}
+                    alignmentBaseline="central"
+                >
+                    ELEC
+
+                </text>
+                <Triangle x={500} y={y + 180} colour={elecTriangleColour} fill={elecTriangleFill} orientation={-90} />
+                <line className={elecRightFormat} x1={490} y1={y + 180} x2={500} y2={y + 180} />
+
+                <text className="Psi" x={205} y={y + 70} alignmentBaseline="central">PSI</text>
+                <text className="Psi" x={395} y={y + 70} alignmentBaseline="central">PSI</text>
+
+            </svg>
+        </>
+    );
+};
+
+type HydSysProps = {
+    title: string,
+    pressure: number,
+    hydLevel: number,
+    x: number,
+    y: number,
+    fireValve: number,
+    pumpPBStatus: number,
+    yellowElectricPumpStatus: number
+}
+
+const HydSys = ({ title, pressure, hydLevel, x, y, fireValve, pumpPBStatus, yellowElectricPumpStatus } : HydSysProps) => {
+    const [hydLevelLow, setHydLevelLow] = useState(false);
+    const lowPressure = 1450;
+    const pressureNearest50 = Math.round(pressure / 50) * 50 >= 100 ? Math.round(pressure / 50) * 50 : 0;
+
+    const [greenPumpActive] = useSimVar('L:A32NX_HYD_GREEN_EDPUMP_ACTIVE', 'boolean', 500);
+    const [yellowPumpActive] = useSimVar('L:A32NX_HYD_YELLOW_EDPUMP_ACTIVE', 'boolean', 500);
+    const [bluePumpActive] = useSimVar('L:A32NX_HYD_BLUE_EPUMP_ACTIVE', 'boolean', 500);
+
+    const [greenPumpLowPressure] = useSimVar('L:A32NX_HYD_GREEN_EDPUMP_LOW_PRESS', 'boolean', 500);
+    const [yellowPumpLowPressure] = useSimVar('L:A32NX_HYD_YELLOW_EDPUMP_LOW_PRESS', 'boolean', 500);
+    const [bluePumpLowPressure] = useSimVar('L:A32NX_HYD_BLUE_EPUMP_LOW_PRESS', 'boolean', 500);
+
+    function checkPumpLowPressure(pump) {
+        switch (pump) {
+        case 'GREEN':
+            return (greenPumpLowPressure && greenPumpActive) || !greenPumpActive;
+        case 'BLUE':
+            return (bluePumpLowPressure && bluePumpActive) || !bluePumpActive;
+        case 'YELLOW':
+            return (yellowPumpLowPressure && yellowPumpActive) || !yellowPumpActive;
+        default:
+            return 1;
+        }
```

Comment:
Due to recent brain problems I can't figure out if that low pressure local check is needed or not.
LOW_PRESS simvars are currently build from wasm using LO_PRESS= is_active && pressure_is_low. Would that be enough to just use those directly?

Reply:
It might. Let me check tonight.